docs/api-status.json

```diff
@@ -1159,6 +1159,12 @@
         "comment": "Exec executes an OSD class method on an object.\nSee rados_exec() in the RADOS C api documentation for a general description.\n\nImplements:\n\n\tvoid rados_write_op_exec(rados_write_op_t write_op,\n\t                         const char *cls,\n\t                         const char *method,\n\t                         const char *in_buf,\n\t                         size_t in_len,\n\t                         int *prval)\n",
         "added_in_version": "v0.29.0",
         "expected_stable_version": "v0.31.0"
+      },
+      {
+        "name": "Conn.GetAddrs",
+        "comment": "GetAddrs wraps the function to get the addresses of\nthe RADOS session, suitable for blocklisting.\n\nImplements:\n\n\tint rados_getaddrs(rados_t cluster, char **addrs)\n",
```

Comment:
This generated JSON needs to be updated now that the code comment is changed. Try using git to revert the change to *just this file* and then re-run the make command to regenerate the JSON. (then squash it all back together :-) )

Reply:
If you need more instruction on all that feel free to ask.